src/functions/sendCandidateResults/application/templates/appeal-your-test.ts

```diff
@@ -18,9 +18,9 @@ export enum AppealUrls {
 
 export const howToAppealTemplate = (url: string, category?: TestCategory, riding?: boolean): string => {
   return  `
-# How to appeal your ${riding ? 'motorcycle' : 'driving'} test
+# How to appeal your ${category === TestCategory.ADI3 ? 'test' : category === TestCategory.SC ? 'standards check' : (riding ? 'motorcycle test' : 'driving test')} 
 
-You can appeal to a court if you think your driving examiner did not follow the law about how they must carry out ${category === TestCategory.ADI3 ? 'ADI qualifying tests' : category === TestCategory.SC ? 'ADI standards checks' : 'driving tests'}.
+You can appeal to a court if you think your ${(category === TestCategory.ADI3 || category === TestCategory.SC) ? '' : 'driving'} examiner did not follow the law about how they must carry out ${category === TestCategory.ADI3 ? 'ADI qualifying tests' : category === TestCategory.SC ? 'ADI standards checks' : 'driving tests'}.
```

Comment:
Just being pedantic, but you could do here:

${[TestCategory.ADI3, TestCategory.SC].includes(category) ? '' : 'driving'}